tsconfig.json

```diff
@@ -1,21 +1,14 @@
 {
     "compilerOptions": {
+        "esModuleInterop": true,
+        "skipLibCheck": true,
+        "outDir": "./dist",
+        "sourceMap": false,
+        "allowJs": true,
+        "strictNullChecks": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
```

Comment:
Any reason you can't use `strict = true` instead of this subset? `strictNullChecks` is usually the most "annoying" thing, so if that's cool, usually going with `strict` is nicer. It will also make things potentially less loose in the future when they add more strictness checks over time.

Reply:
Changed to `strict`